net/frr/src/opnsense/service/templates/OPNsense/Quagga/frr

```diff
@@ -11,6 +11,8 @@ if helpers.exists('OPNsense.quagga.bgp.enabled') and OPNsense.quagga.bgp.enabled
 if helpers.exists('OPNsense.quagga.ospf6.enabled') and OPNsense.quagga.ospf6.enabled == '1' %} ospf6d{% endif %}{%
 if helpers.exists('OPNsense.quagga.ripng.enabled') and OPNsense.quagga.ripng.enabled == '1' %} ripngd{% endif %}{%
 if helpers.exists('OPNsense.quagga.isis.enabled') and OPNsense.quagga.isis.enabled == '1' %} isisd{% endif %}"
+frr_carp_demote="{%
+if not helpers.empty('OPNsense.quagga.ospf.carp_demote') %} ospfd{% endif %}"
```

Comment:
Is there a reason for the trailing space?

Reply:
not really, it looked easier to read at the time, but looking at it again, I think it's better to remove it.